### DISCUSSION TITLE
OTX integration with Suricata

### DISCUSSION THREAD

**Discussion by faisalusuf** [Security-Onion-Solutions/securityonion#4026] [Unsupported Versions]
Hi,

While looking at the docs, there is a guide available which explains how to integrate OTX feeds but the IDS in this guide is Zeek. Is this possible to use Suricata and integrate OTX as per guide:

Reference: `https://docs.securityonion.net/en/2.3/alienvault-otx.html`

Thanks
Faisal.

**Comment by faisalusuf** [comment]
Hello,

Anyone tried OTX with Suricata on SO?

Thanks

**Reply by weslambert** [reply]
I have not.  Similarly, there is a script to pull down NIDS rules for Suricata from MISP here:
https://github.com/weslambert/securityonion-misp
